marconi-mamba/README.md

```diff
@@ -0,0 +1,35 @@
+marconi-mamba
```

Comment:
@raduom @koslambrou
This is a draft readme for now. For now, it porvides some instruction and a rest-client script to see how the json-rpc http server is coming along.
At this point, we are just serving a moc-txOtuRef